src/main/resources/api/data-repository-openapi.yaml

```
@@ -2828,6 +2828,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/datasets/{id}/enumerateSnapshotRequests:
+    post:
```

I think this should be `get`, is there a reason why it needs to be `post`?
```suggestion
    get:
```